MiniApp/Classes/Protocols/MiniAppUserInfoDelegate.swift

```diff
@@ -18,4 +21,18 @@ public extension MiniAppUserInfoDelegate {
     func getProfilePhoto() -> String? {
         return nil
     }
+
+    func getAccessToken(miniAppId: String, completionHandler: @escaping (Result<MATokenInfo, MASDKCustomPermissionError>) -> Void) {
+        completionHandler(.failure(.failedToConformToProtocol))
+    }
+}
+
+public class MATokenInfo: Codable {
+    let token: String
+    let validUntil: Date
```

Comment:
Is `Date` the correct format for this? The JS bridge is expecting a `number`. I think it's okay to have the Host App provide a `Date`, but can we convert it to the correct number format before sending to the bridge?

Reply:
Updated now